File: Beecrowd/2058 Triângulos e Polígonos Regulares/main.cpp
#include <iostream>

using namespace std;

int main(){
    int n = 0, res;

    while(n < 3){
        cin >> n;
        
        if(n >= 3){
            res = n - 2;

            cout << res << endl;
        }
    }

    return 0;
}

File: Beecrowd/2058 Triângulos e Polígonos Regulares/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        int n, res;

        n = sc.nextInt();

        res = n - 2;

        System.out.println(res);

        sc.close();
    }
}